meeting-notes/2019-Oct-02.md

```diff
@@ -0,0 +1,64 @@
+## Falco Office Hours
+
+### MC
+
+Leonardo Di Donato
+
+### Who joined
+
+- Leonardo Di Donato (@leodido)
+- Lorenzo Fontana (@fntlnz)
+- _..._
+
+### This week highlights
+
+- Leo and Lorenzo found Falco vulnerability and fixed it
+- Leo and Lorenzo released 0.17.1
+- Almost ready to release 0.18.0
+
+### Proposals / Discussions
+
+- Release [0.18.0](https://github.com/falcosecurity/falco/milestone/1)
```

Comment:
We ask as first point who wants to run next week.

Reply:
@kris-nova booked herself for next on slack!

I'll notify it during the call